src/index.ts

```diff
@@ -12,6 +13,7 @@ const controller = new SafeAreaController();
 export * from './definitions';
 export { SafeArea };
 export { controller as SafeAreaController };
+export { SafeAreaElement, registerSafeAreaElement };
```

Comment:
@jjang16 tagging just in case, rest all the changes seems fine to me, I'm just not sure about this one, though it's not big, I just don't see much use case for this export.

Reply:
Exporting SafeAreaElement is just for keeping things transparent.

Someone might
1. Use the class type definition
2. Extend the class
3. Bypass the register function and use their own implementation

It also clarifies all the features this library exposes within index.ts : SafeArea, SafeAreaController, and SafeAreaElement.

Thank you.